src/components/LoanFinding/QuickFiltersSection.vue

```diff
@@ -142,13 +178,17 @@ export default {
 				this.loans = this.backupLoans;
 			}
 		},
-		resetFilters() {
+		async resetFilters() {
```

Comment:
When you click reset, the see more button gets the follow URL: `/lend/filter?location=`

Reply:
This was caused by the `filterPageUrl` issue above, should be ok now^